airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2566 @@
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/stories_compact_partition_router"
+
+  workspaces_stream:
+    $ref: "#/definitions/base_stream"
+    name: "workspaces"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/workspaces_schema"
+    $parameters:
+      path: "/workspaces"
+      extractor_path: "data"
+
+  workspaces_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/workspaces_stream"
+        parent_key: "gid"
+        partition_field: "workspace_gid"
+
+  tags_stream:
+    $ref: "#/definitions/base_stream"
+    name: "tags"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tags_schema"
+    $parameters:
+      path: "/workspaces/{{ stream_slice['workspace_gid'] }}/tags"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        $ref: "#/definitions/workspaces_partition_router"
+
+  users_stream:
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/users_schema"
+    $parameters:
+      path: "/users"
+      extractor_path: "data"
+
+  users_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/users_stream"
+        parent_key: "gid"
+        partition_field: "users_gid"
+
+  teams_stream:
+    $ref: "#/definitions/base_stream"
+    name: "teams"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/teams_schema"
+    $parameters:
+      path: "/organizations/{{ stream_slice['workspace_gid'] }}/teams"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      partition_router:
+        $ref: "#/definitions/workspaces_partition_router"
+
+  teams_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/teams_stream"
+        parent_key: "gid"
+        partition_field: "team_gid"
+
+  team_memberships_stream:
+    $ref: "#/definitions/base_stream"
+    name: "team_memberships"
+    primary_key: "gid"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/team_memberships_schema"
+    $parameters:
+      path: "/teams/{{ stream_slice['team_gid'] }}/team_memberships"
+      extractor_path: "data"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/teams_partition_router"
+
+  attachments_compact_stream:
+    $ref: "#/definitions/base_stream"
+    name: "attachments_compact"
+    primary_key: "gid"
+    schema_loader:
```

Comment:
Could you please explain about the fix, 
What should be done to make this fixed,
I am kinda new to this type of error
@askarpets

Reply:
I've removed extractor_path variable

Reply:
Is this an error? those fields are mandate for connector to work isn't i

Reply:
Those fields should not be exposed when running `discover` command. We need some time to find the solution. I will get back to you once it's found

Reply:
To avoid these extra fields, you should update your schemas definitions to use `type: object` instead of `type: ["null", object]`, but this should only be applied for the schema's type itself, not objects inside it

Reply:
Done